### DISCUSSION TITLE
Slovak language

### DISCUSSION THREAD

**Discussion by maciak01** [sumatrapdfreader/sumatrapdf#4242] [Q&A]
Hello, 
I just installed the latest version and I can't change the language to Slovak. Can you help me with that well? Thank you.

**Comment by GitHubRulesOK** [comment]
The Language ability is often version dependant based on coverage at the time. 
If there was low coverage it is not included in that version.

Current Pre-release 3.6.16121
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/4a41cd8d-4959-4a15-a6f8-f17792f90ae3)

**Comment by maciak01** [comment]
Thank you. :-)
